packages/grid/x-data-grid/src/components/menu/GridMenu.tsx

```diff
@@ -76,9 +78,15 @@ const GridMenu = (props: GridMenuProps) => {
       (prevTarget.current as HTMLElement).focus();
     }
 
+    if (open) {
+      apiRef.current.publishEvent('menuOpen', { target, open });
+    } else {
+      apiRef.current.publishEvent('menuClose', { target, open });
+    }
```

Comment:
The arguments are almost the same. We could have only one `publishEvent` and vary the event name. I think the `open` param could be removed since the event name already tells the current state.

Reply:
Updated